### DISCUSSION TITLE
Don't enable emulators by default.

### DISCUSSION THREAD

**Discussion by luc122c** [vuejs/vuefire#1393] [Ideas]
After upgrading to the latest version, I got the following console warning:
> **Warning**
>  You enabled emulators but there is no emulators key in your firebase.json file. Emulators will not be enabled. 

This is in a project where I am not using the emulators. After taking a look through the code, I noticed that as of #1392, [emulators are enabled by default](https://github.com/vuejs/vuefire/blob/66bd12c0a431b0874c9004271966aeb7505c1777/packages/nuxt/src/module.ts#L39). 

Even though it is only a warning, I feel that since support for emulators is new enabling them by default is a breaking change. I'd suggest changing the default to `false`. I'd be more than happy to contribute a section on enabling emulator support to the vuefire docs.

**Comment by posva** [comment]
Emulators should be active only if they are defined in your `firebase.json`, it was a mistake that they were fully enabled by default but it should be fixed now! Thanks for reaching out!
